examples/standalone-enhanced-org-policy/centralized_logging.tf

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
consider renaming this file to "harness.tf" to match the other standalone and to indicate this is optional